src/Catalog/Registration/RegistrationMakerCatalogItem.cs

```diff
@@ -155,6 +157,43 @@ private Uri GetPackageContentAddress()
             return _packageContentAddress;
         }
 
+        private string GetLicenseUrl()
+        {
+            INode subject = _catalogItem.CreateUriNode(_catalogUri);
+
+            string id = _catalogItem.GetTriplesWithSubjectPredicate(subject, _catalogItem.CreateUriNode(Schema.Predicates.Id)).FirstOrDefault().Object.ToString();
+            string version = _catalogItem.GetTriplesWithSubjectPredicate(subject, _catalogItem.CreateUriNode(Schema.Predicates.Version)).FirstOrDefault().Object.ToString();
```

Comment:
This is full version, I believe. We should only be using normalized version in the URL. See what `packageContent` is doing.

Reply:
normalize the version now.